client/src/main/kotlin/no/nav/su/se/bakover/client/dokarkiv/JournalpostRequest.kt

```diff
@@ -75,23 +71,46 @@ sealed class Journalpost {
         override val journalpostType: JournalPostType = JournalPostType.UTGAAENDE
         override val kanal: String? = null
         override val journalfoerendeEnhet: String = "4815"
-        override val dokumenter: List<JournalpostDokument> = vedtakInnhold.toJournalpostDokument()
-
-        private fun VedtakInnhold.toJournalpostDokument() = listOf(
+        override val dokumenter: List<JournalpostDokument> = listOf(
             JournalpostDokument(
-                tittel = "Vedtaksbrev for soknad om supplerende stønad",
+                tittel = tittel,
                 dokumentKategori = DokumentKategori.VB,
                 dokumentvarianter = listOf(
-                    DokumentVariant(
-                        filtype = "PDFA",
-                        fysiskDokument = Base64.getEncoder().encodeToString(pdf),
-                        variantformat = "ARKIV"
-                    ),
-                    DokumentVariant(
-                        filtype = "JSON",
-                        fysiskDokument = Base64.getEncoder()
-                            .encodeToString(objectMapper.writeValueAsString(this).toByteArray()),
-                        variantformat = "ORIGINAL"
+                    DokumentVariant.Arkiv(fysiskDokument = Base64.getEncoder().encodeToString(pdf)),
+                    DokumentVariant.OriginalJson(
+                        fysiskDokument =
+                            Base64.getEncoder().encodeToString(objectMapper.writeValueAsString(vedtakInnhold).toByteArray())
+                    )
+                )
+            )
+        )
+    }
+
+    data class lukketSøknadJournalpostRequest(
+        val person: Person,
+        val pdf: ByteArray,
+        val sakId: UUID,
+        val lukketSøknadBrevinnhold: LukketSøknadBrevinnhold,
+    ) : Journalpost() {
+        override val tittel: String = "Bekrefter at søknad er trukket"
+        override val avsenderMottaker: AvsenderMottaker = AvsenderMottaker(
+            id = person.ident.fnr.toString(),
+            navn = søkersNavn(person)
+        )
+        override val sak: Fagsak = Fagsak(sakId.toString())
+        override val bruker: Bruker = Bruker(id = person.ident.fnr.toString())
+        override val journalpostType: JournalPostType = JournalPostType.UTGAAENDE
+        override val kanal: String? = null
+        override val journalfoerendeEnhet: String = "4815"
```

Comment:
Er det kun her vi bruk enheten? Hvis ikke kan den trekkes ut i en const

Reply:
Denne og for vedtak. høres ut som god plan å trekkes ut.